ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1316,6 +1316,15 @@ literals:
 - outside_condensing_loop_return_water_isolation_valve_percentage_sensor
 - outside_condensing_loop_return_water_temperature_sensor
 
+# Rodent Repellent system
+- wave_pulserate_sensor
+- wave_frequency_sensor
+- wave_band_mode:
+  - BANDA
+  - BANDB
+  - BANDC
+  - BANDD
+- total_transducer_failed_counter
```

Comment:
please provide some context: what device does this point belong to? based on the name, i would expect this to belong to a gateway device, counting the total number of transducers that are failed. i dont think you would actually model that gateway separately, though... each transducer would generally be its own device and it would have its own `failed_alarm`.